not-to-release/sources/answers/20111106015552AAj6rCu_ans.xml.conllu

```diff
@@ -17,7 +17,7 @@
 13	from	from	ADP	IN	_	7	obl	7:obl	_
 14	in	in	ADP	IN	_	15	case	15:case	_
 15	Invercargill	Invercargill	PROPN	NNP	Number=Sing	1	obl	1:obl:in	_
-16	New	New	PROPN	NNP	Number=Sing	17	compound	17:compound	_
+16	New	New	ADJ	NNP	Degree=Pos	17	amod	17:amod	_
```

Comment:
NB in GUM New Zealand and New York are PROPN + flat, based on the idea that New is now completely opaque and it's just a flat name. Would you be OK adopting that?

Reply:
I don't like having to make a semantic distinction. Is General Motors semantically compositional? Good Friday? Old Faithful? Seems like a can of worms.